app/src/androidTest/java/com/github/polypoly/app/map/GameActivityTest.kt

```diff
@@ -157,9 +90,66 @@ class GameActivityTest: PolyPolyTest(true, false) {
         // TODO: Replace by future MAX_BET or similar
         composeTestRule.onNodeWithTag("betInput").performTextInput("3000")
         composeTestRule.onNodeWithTag("confirmBetButton", true).performClick()
-        composeTestRule.onNodeWithTag("betDialog", true).assertDoesNotExist()
+        // composeTestRule.onNodeWithTag("betDialog", true).assertDoesNotExist()
     }
-    
+
+    // While it may be better for grades to have a test for each component,
```

Comment:
for grades we want good tests, and sometimes it makes total sense to merge tests, that's why comments are helpful :-)